stingray/crossspectrum.py

```diff
@@ -565,9 +565,10 @@ def __init__(self, lc1=None, lc2=None, segment_size=None,
         """
         self.type = "crossspectrum"
 
-        if segment_size is not None:
```

Comment:
If segment size must be specified, does it not make more sense not to have a default for this? 
Move `segment_size` into the spot as the first parameter and make it mandatory?